spec/support/shared_examples/lorekeeper_loggers.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'Lorekeeper loggers' do
+  let(:message) { 'All was well.' }
+
+  it 'respond to write' do
+    logger.public_send(:write, message)
```

Comment:
I don't think you need to use `public_send` here
```suggestion
    logger.write(message)
```

also, the `write` method is not doing the same in all loggers. instead of having this shared spec, how about adding specs FastLogger and JSONLogger and checking the actual logic?

Reply:
indeed 🤦‍♀️